b2share/modules/files/permissions.py

```diff
@@ -25,7 +25,7 @@
 from functools import partial
 
 from invenio_access.permissions import (
-    superuser_access, ParameterizedActionNeed, DynamicPermission
+    superuser_access, ParameterizedActionNeed, Permission
```

Comment:
TODO: Change back to Dynamicpermission

https://github.com/EUDAT-B2SHARE/b2share/pull/1842/files#r594444350